File: backend-example-dotnet/src/application/test/TestIocClass2.cs
namespace BackendExample.Application.Test
{
    /// <summary>
    /// IoC测试类2
    /// </summary>
    public class TestIocClass2
    {
        private readonly TestIocClass3 clss3;

        public TestIocClass2(TestIocClass3 clss3)
        {
            this.clss3 = clss3;
        }
    }
}

File: backend-example-java/src/main/java/com/example/backend/application/test/TestIocClass2.java
package com.example.backend.application.test;

import org.springframework.stereotype.Component;

/**
 * IoC测试类2
 */
@Component
public class TestIocClass2 {

    private final TestIocClass3 class3;

    public TestIocClass2(TestIocClass3 class3) {
        this.class3 = class3;
    }
}
